Events/ShipTargetedEvent.cs

```diff
@@ -0,0 +1,85 @@
+﻿using EddiDataDefinitions;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace EddiEvents
+{
+    public class ShipTargetedEvent : Event
+    {
+        public const string NAME = "Ship targeted";
+        public const string DESCRIPTION = "Triggered when the player selects a target";
+        public const string SAMPLE = "{\"timestamp\":\"2018-05-09T23:19:49Z\",\"event\":\"ShipTargeted\",\"TargetLocked\":true,\"Ship\":\"adder\",\"ScanStage\":3,\"PilotName\":\"$npc_name_decorate:#name=Phoenix;\",\"PilotName_Localised\":\"Phoenix\",\"PilotRank\":\"Competent\",\"ShieldHealth\":100.000000,\"HullHealth\":100.000000,\"Faction\":\"Union Cosmos\",\"LegalStatus\":\"Lawless\",\"Subsystem\":\"$int_powerplant_size3_class3_name;\",\"Subsystem_Localised\":\"Power Plant\",\"SubsystemHealth\":100.000000}";
+        public static Dictionary<string, string> VARIABLES = new Dictionary<string, string>();
+
+        static ShipTargetedEvent()
+        {
+            VARIABLES.Add("targetlocked", "True when a ship has been targeted. False when a target has been lost/deselected");
+            VARIABLES.Add("ship", "the model of the ship");
+            VARIABLES.Add("scanstage", "the stage of the ship scan (e.g. 0, 1, 2, or 3)");
+            VARIABLES.Add("name", "The name of the pilot");
+            VARIABLES.Add("rank", "The rank of the pilot");
+            VARIABLES.Add("faction", "The faction of the pilot");
+            VARIABLES.Add("legalstatus", "The legal status of the pilot");
+            VARIABLES.Add("shieldhealth", "The health of the shields");
+            VARIABLES.Add("hullhealth", "The health of the hull");
+            VARIABLES.Add("subsystem", "The subsystem targetted");
+            VARIABLES.Add("subsystemhealth", "The health of the subsystem targetted");
```

Comment:
typo. Should be: `targeted.`

Reply:
Lol, yeah. If you look in the Journal manual, it's misspelled there too. Fixed.

Reply:
I appreciate your thoughts, TK. Lol, we do have a bit different design philosophy... and that's a good thing. :-)

I tend to go with the "less is more' philosophy.

Additionally, if users are going to be crafting their own scripts, it behooves us to encourage them to crack open the Journal Manual and get a more intimate understanding of what makes the Journal tick. Everything they need to know about the 'Ship Targeted' event is in the Journal Manual. As a compromise, I will copy the stage description in the Journal Manual and copy it as comments to the event script.